doc/guides/user/quick_start.rst

```diff
@@ -0,0 +1,316 @@
+===================
+pydicom quick start
+===================
+
+Welcome to the quick start guide for *pydicom*.
+
+*pydicom* is an MIT licensed, open source `Python <https://www.python.org/>`_ library for
+creating, reading, modifying and writing `DICOM <https://www.dicomstandard.org/>`_
+:dcm:`Data Sets<part05/chapter_7.html>` (*datasets* for short) and
+:dcm:`File-sets<part10/chapter_8.html>`. It can also convert
+the imaging and waveform data in certain dataset types to a `NumPy <https://www.numpy.org>`_
+``ndarray`` (and back again), as long as suitable optional packages are :doc:`installed
+</guides/user/installation>`.
+
+What is a DICOM dataset?
+========================
+
+A DICOM dataset represents an instance of a real world object, such as a single image slice from
+a CT scan acquisition. Each dataset is made up of a collection of :dcm:`Data Elements
+<part05/chapter_7.html#sect_7.1>`, with each Data Element representing an *attribute* of
+the object. A Data Element is itself made of a unique identifier called the *Element Tag*,
```

Comment:
I wouldn't say `unique` here, since the same tag can appear multiple times in sequences.  Just `identifier` would be more correct and just as clear.

Reply:
The tag is unique to the element, though.